### ISSUE TITLE
Multi-Language Spellchecker doesn't work.

### ISSUE THREAD

**Issue by ccorcos** [electron/electron#15202]
* Output of `node_modules/.bin/electron --version`: v2.0.5
* Operating System (Platform and Version): MacOS

The following will not check spelling for words with russian letters.

```ts
electron.webFrame.setSpellCheckProvider({
    language: "en",
    ...
})
```

The following will not check spelling for words with english letters.

```ts
electron.webFrame.setSpellCheckProvider({
    language: "ru",
    ...
})
```

The following will not check spelling for words with russian letters.

```ts
electron.webFrame.setSpellCheckProvider({
    language: "en,ru",
    ...
})
```

At the very least, I would hope that `en,ru` should work.

**Comment by ccorcos** [comment]
The example you showed me is still just one language at a time. 

I'm using [node-cld](https://github.com/dachev/node-cld) which can detect multiple languages in the same string. For example `Предложение на русскм языке без ошиб hello world. does this work well? ` returns Russian and English. 

Using [node-spellchecker](https://github.com/atom/node-spellchecker) I can check if a word is misspelled in either language, but the electron spellchecking api is not checking all words so I'm fundamentally limited by Electron now.

Honestly, I'm not sure why I have to provide a language to setSpellCheckProvider. Ideally it would always check all words and leave it up to the developer. 

Please reconsider opening this issue. Chrome spellchecks multiple languages at the same time very gracefully. Ideally we can have a similar functionality in electron apps.

**Comment by ccorcos** [comment]
This looks like the heart of the issue:

https://github.com/electron/electron/blob/6f91af93433df4e9c0e7fb592e5b2dcb0b1c7888/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.h#L37-L41

**Comment by ccorcos** [comment]
Thanks for the breakdown. Looks like you're ahead of me. Are you using your hunspell library in an application? Mind if I ask which app?

**Comment by kwonoj** [comment]
> Are you using your hunspell library in an application

yes. https://github.com/kwonoj/electron-hunspell uses https://github.com/kwonoj/hunspell-asm in behind, and I'm using https://github.com/kwonoj/cld3-asm for language detection.

> Mind if I ask which app

Sorry, allow me to not share which app in here.
